src/_lib/sass/src/_email-input.scss

```diff
@@ -47,6 +47,8 @@ nw-email-input {
         &:focus {
             outline: none;
             box-shadow: none;
+            background-color: transparent;
+            font-size: 12px;
```

Comment:
Fix for the small visual issue with the input having a grey bg colour and the font being too large.